easybuild/easyconfigs/g/GSL/GSL-1.16-goolf-1.5.14.eb

```diff
@@ -0,0 +1,18 @@
+easyblock = 'ConfigureMake'
+
+name = 'GSL'
+version = '1.16'
+
+homepage = 'http://www.gnu.org/software/gsl/'
+description = """The GNU Scientific Library (GSL) is a numerical library for C and C++ programmers.
+The library provides a wide range of mathematical routines such as random number generators, special functions and least-squares fitting."""
+
+toolchain = {'name': 'goolf', 'version': '1.5.14'}
+toolchainopts = {'opt': True, 'optarch': True, 'unroll': True, 'pic': True}
+
+source_urls = ['http://ftpmirror.gnu.org/gsl/']
```

Comment:
@scintimandrion: please fix this tiny remark, and let us know if you did by issuing a comment